app/backend/approaches/chatreadretrieveread.py

```diff
@@ -117,3 +114,66 @@ def get_chat_history_as_text(self, history: Sequence[dict[str, str]], include_la
             if len(history_text) > approx_max_tokens*4:
                 break    
         return history_text
+    
+    # Generate messages needed for chat Completion api
```

Comment:
Please move comment inside function, using standard docstring formatting, and capitalization.

"""Generate messages needed for Chat Completion API"""